src/components/Slider/__tests__/__snapshots__/slider-non-ranged-tests.jest.js.snap

```diff
@@ -56,12 +56,12 @@ Snapshot Diff:
 +           class="monday-style-dialog-content-component top edge-center has-tooltip monday-style-animation-expand-appear-done monday-style-animation-expand-enter-done"
 +         >
 +           <div
-+             class="monday-style-tooltip monday-style-tooltip-dark padding-size-undefined"
++             class="mondayStyleTooltip monday-style-tooltip mondayStyleTooltipDark monday-style-tooltip-dark paddingSizeUndefined padding-size-undefined"
```

Comment:
Maybe it's ok but "paddingSizeUndefined" looks to me like maybe there is an issue with converting the padding size to class name?
Worth double check?

Reply:
It's not a trouble of css modules conversion - cause previously it was like this `padding-size-undefined`, but I see how  it can be refactored - will do

Reply:
Refactored